src/CustomerProfile.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+
+const CustomerProfile = ({ customerId }) => {
+  const [customerProf, setCustomerProf] = useState({});
+  console.log(`https://cyf-react.glitch.me/customers/${customerId}/`);
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/customers/${customerId}/`)
+      .then(res => res.json())
+      .then(data => {
+        setCustomerProf(data);
+        console.log(data);
+      });
+  }, [customerId]);
+  if (!customerProf) return "Loading...";
```

Comment:
`customerProf` variable is initialised with an empty object so it will always be `truthy` and "Loading" will be never displayed. If you haven't heard about truthy and falsy values in JavaScript - you can read more [here](http://adripofjavascript.com/blog/drips/truthy-and-falsy-values-in-javascript.html). Because an empty object is truthy, you will need to explicitly check that it's empty, for example by checking the length of object keys array:
```
if (!Object.keys(customerProf).length)
```
Also "Loading" message should be an HTML element.